src/components/Error.js

```diff
@@ -23,7 +23,7 @@ class Error extends React.Component {
             Let Us Know!
           </Button>
           &nbsp;
-          <Link to="/">
+          <Link to={this.props.returnTo ? this.props.returnTo : "/"}>
```

Comment:
Pretty sure we can just drop in:

```suggestion
          <Link to={this.props.returnTo || "/"}>
```